sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/ParDoTest.java

```diff
@@ -1001,28 +1005,33 @@ public void verifyDeterministic() {}
   }
 
   private static class TaggedOutputDummyFn extends DoFn<Integer, Integer> {
+    private TupleTag<Integer> mainOutputTag;
     private TupleTag<TestDummy> dummyOutputTag;
-    public TaggedOutputDummyFn(TupleTag<TestDummy> dummyOutputTag) {
+    public TaggedOutputDummyFn(TupleTag<Integer> mainOutputTag,
+                               TupleTag<TestDummy> dummyOutputTag) {
+      this.mainOutputTag = mainOutputTag;
       this.dummyOutputTag = dummyOutputTag;
     }
 
     @ProcessElement
-    public void processElement(ProcessContext c) {
-      c.output(1);
-      c.output(dummyOutputTag, new TestDummy());
+    public void processElement(MultiOutputReceiver r) {
```

Comment:
Kind of hard to quickly find, since there's a bunch of extra reformatting, but we should leave at least one explicit test of `ProcessContext` as an automatic DoFn parameter.